README.md

```diff
@@ -1,6 +1,6 @@
 # grpc-java-api-checker
 
-[![Build Status](https://travis-ci.org/grpc/grpc-java-api-checker.svg?branch=master)](https://travis-ci.org/grpc/grpc-java-api-checker)
+[![GitHub Actions Testing](https://github.com/grpc/grpc-java-api-checker/actions/workflows/testing.yml/badge.svg)](https://github.com/grpc/grpc-java-api-checker/actions/workflows/testing.yml)
```

Comment:
I think we need `?branch=master` as well. https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge#using-the-branch-parameter

Reply:
Thank you for pointing it out.

Reply:
We need the ?branch on the svg as well.

Reply:
:smile: It's times like this that you begin realizing it is amazing anything works at all.

Reply:
That's true...